website/docs/reference/dbt-jinja-functions/cross-database-macros.md

```diff
@@ -0,0 +1,537 @@
+---
+title: "cross-database macros"
+id: "cross-database-macros"
+---
+
+# Overview
+
+These macros make it easier for package authors (especially those writing modeling packages) to implement cross-database
+compatibility. In general, you should not need these macros in your own dbt project (unless it is a package or you otherwise desire cross-database compatibility).
```

Comment:
👍 Glad to see this context here!

The way I'd put this in my head:
- If you maintain a package, by using these macros, rather than a specific database's SQL syntax, your package is more likely to work on other adapters.
- If you maintain an adapter, by implementing (and testing) these macros, your adapter is more likely to support more packages.
- If you're an end user, more packages and adapters are likely to "just work" for you, without you having to do anything.
- Win-win!